docs/specs/component.md

```diff
@@ -208,11 +208,14 @@ implement since errors are specific to the component.
 
 * Properties
   * `error` - The specifics of the error condition, such as system error code, etc.
-  * `stage` - The stage at which the error occurred. MUST be one of
-    `receiving`, `processing`, or `sending`. This MAY be omitted from
-    being represented explicitly in the event data when the error may
-    only happen at one stage, but MUST be included in the emitted logs
-    and metrics as if it were present.
+  * `error_type` - The type of error condition. This MUST be one of the types
+    listed in the `error_type` enum list in the cue docs. TODO: List them here?
```

Comment:
🤔 I guess we could just link to the cue doc here. I'm not seeing another alternative at the moment aside from duplicating them.

Reply:
Just a note to resolve this TODO